### PR TITLE
[ID] Retranslate `Client/Keyboard_shortcuts`

### PR DESCRIPTION
**Pull request by Nivalyx**
a complete overhaul of the Indonesian localization of the http://osu.ppy.sh/wiki/en/Client/Keyboard_shortcuts wiki page.

i don't think i've ever realized it before but ngl upon working on this i found out that -- out of the Indonesian locale pages that i've been cleaning up/re-translating so far--  perhaps this is the one that goes all over the place the most 😰  

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by ZeroPyrozen — DISMISSED** [review]

File: wiki/Client/Keyboard_shortcuts/id.md [hunk 1]
```diff
@@ -7,96 +7,96 @@ tags:
...
-| `Shift` + `F1` | Membuka [osu!status](https://twitter.com/osustatus) di web browser kamu. (Hanya ketika terjadi kesalahan koneksi.) |
-| `Ctrl` + `O` | Membuka sidebar pengaturan. Tidak bisa membuka offset wizard jika tidak berada di menu utama. |
+| `F9` | Menampilkan/menyembunyikan jendela obrolan dan daftar pengguna. |
+| `F10` | Mengaktifkan/menonaktifkan tombol-tombol mouse (kecuali roda mouse, yang akan selalu aktif sepanjang waktu). |
+| `F12` | Mengambil tangkapan layar (*screenshot*). Klik pada dialog yang muncul untuk membuka folder Screenshots. |
+| `Shift` + `F12` | Mengambil tangkapan layar (*screenshot*) untuk kemudian diunggah ke server osu!. Perintah ini akan menyimpan tangkapan layar (*screenshot*) secara lokal pada folder Screenshots, mengunggahnya, dan menampilkannya pada *browser*. |
+| `Insert` | Tombol ini merupakan [tombol bos (*boss key*)](https://en.wikipedia.org/wiki/Boss_key). Pada saat ditekan, tombol ini akan meminimalkan osu! akan ke area notifikasi dan mematikan suara lagu. |
+| `Shift` + `F1` | Membuka laman [osu!status](https://twitter.com/osustatus) pada *browser*. (Hanya dapat digunakan pada saat osu! gagal terhubung) |
```
Comment (Walavouchey) [inline wiki/Client/Keyboard_shortcuts/id.md:33]: missing period?

```suggestion
| `Shift` + `F1` | Membuka laman [osu!status](https://twitter.com/osustatus) pada *browser*. (Hanya dapat digunakan pada saat osu! gagal terhubung.) |
```

File: wiki/Client/Keyboard_shortcuts/id.md [hunk 2]
```diff
@@ -7,96 +7,96 @@ tags:
...
-| `E` dua kali | Masuk mode edit. |
-| `O` | Membuka sidebar pengaturan. |
-| `Q` | Penampilkan pemilihan untuk keluar. |
-| `Esc` | Penampilkan pemilihan untuk keluar ketika menu pengaturan dan permainan sudah ditutup. |
-| `F` | Mengganti mode pembatasan frame rate. |
-
-### Jukebox
-
-| Pintasan | Aksi |
+| `Alt` + `Enter` | Mengaktifkan/menonaktifkan mode *fullscreen* |
```
Comment (Walavouchey) [inline wiki/Client/Keyboard_shortcuts/id.md:40]:
```suggestion
| `Alt` + `Enter` | Mengaktifkan/menonaktifkan mode *fullscreen*. |
```

File: wiki/Client/Keyboard_shortcuts/id.md [hunk 3]
```diff
@@ -109,186 +109,186 @@ Pintasan-pintasan ini bekerja di mana pun:
...
+| `Ctrl` + `Panah Atas/Bawah` | Menambah/mengurangi kecepatan pemutaran lagu (*playback speed*) sebesar 25%. |
+| `Ctrl` + `Shift` + `Panah Atas/Bawah` | Menambah/mengurangi kecepatan pemutaran lagu (*playback speed*) sebesar 5%. |
+| `F1` | Beralih ke tab [Compose](/wiki/Client/Beatmap_editor/Compose). |
+| `F2` | Beralih ke tab [Design](/wiki/Client/Beatmap_editor/Design). |
+| `F3` | Beralih ke tab [Timing](/wiki/Client/Beatmap_editor/Timing). |
+| `F4` | Membuka dialog [Song Setup](/wiki/Client/Beatmap_editor/Song_Setup). |
+| `F5` | Memainkan beatmap pada mode uji (*test mode*). |
+| `F6` | Membuka kotak dialog Timing and Control Points. |
+| `Ctrl` + `P` | Menambahkan *uninherited timing point* pada waktu yang ditentukan. Apabila *shortcut* ini dipanggil di luar tab Timing, kotak dialog Timing and Control Points akan ikut terbuka. |
+| `Ctrl` + `Shift` + `P` | Menambahkan *inherited timing point* pada waktu yang ditentukan. Apabila *shortcut* ini digunakan di luar tab Timing, kotak dialog Timing and Control Points akan ikut terbuka. | |
```
Comment (Walavouchey) [inline wiki/Client/Keyboard_shortcuts/id.md:179]: ci somehow didn't catch this

```suggestion
| `Ctrl` + `Shift` + `P` | Menambahkan *inherited timing point* pada waktu yang ditentukan. Apabila *shortcut* ini digunakan di luar tab Timing, kotak dialog Timing and Control Points akan ikut terbuka. |
```

**Comment by Nivalyx** [comment]
applied everything, thanks for the check as always @Walavouchey 👀 

...yeah it's kind of weird that remark doesn't catch that one error that you pointed (but then again both me and Pyrozen was also not aware of that so yeah D:)

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**